packages/sdks/e2e/react-native/package.json

```diff
@@ -6,9 +6,9 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
-    "build": "echo 'no need to build react-native.'",
+    "build": "npx expo export:web",
     "dev": "yarn run web",
-    "serve": "yarn run web",
+    "serve": "npx serve web-build --single",
```

Comment:
We should not use `npx` in our commands:
- `npx` will run using `npm` instead of `yarn`, which can lead to surprises
- if a package is not in the dependencies, `npx` will download it on the spot instead of erroring, leading to surprises

Reply:
makes sense! thanks, let me update it